nautobot/docs/additional-features/jobs.md

```diff
@@ -79,6 +79,16 @@ This is the human-friendly name of your job, as will be displayed in the Nautobo
 #### `description`
 
 A human-friendly description of what this job does.
+This can accept either plain text or markdown-formatted text. It can also be multiple lines like so:
+
+```python
+class ExampleJob(Job):
+    class Meta:
+        description = """
+        #### Title
+        Another line
```

Comment:
I don't think we should be encouraging people to put heading tags in their job descriptions. Can we use a different example, perhaps list formatting or something like that instead?

Also it would be good to call out the first-line-of-the-description-only behavior that will be seen on the job list view.

```
        description = """
            This job does a number of interesting things.

            1. It hacks the Gibson
            2. It immanentizes the eschaton
            3. It's a floor wax *and* a dessert topping
        """
```